finality-aleph/src/network/manager/service.rs

```diff
@@ -96,6 +96,22 @@ impl Config {
     }
 }
 
+type MessageForNetwork<D, M> = (NetworkData<D, M>, DataCommand<<M as Multiaddress>::PeerId>);
+
+pub struct ServiceActions<D: Data, M: Multiaddress> {
+    maybe_command: Option<ConnectionCommand<M>>,
+    data: Vec<MessageForNetwork<D, M>>,
+}
+
+impl<D: Data, M: Multiaddress> ServiceActions<D, M> {
+    fn noop() -> Self {
```

Comment:
I could imagine rewriting it so that we derive `Default` for `ServiceActions` and here simply do:
```rust
fn noop() -> Self { Default::default() }
```

Although one could argue that doing this explicitly is better (and I would probably agree) ;)

Reply:
Then consider that argued. :P

To make this more explicit – I think `Default` implies there is a default value of something, and it would be pretty hard to convince me that a `ServiceAction` should by default be `noop`.

Reply:
Okay, that certainly convinces me, thanks :)